js/sparkswap.js

```diff
@@ -200,8 +210,22 @@ module.exports = class sparkswap extends Exchange {
         throw new ExchangeError ('Not Implemented');
     }
 
-    async commit () {
-        throw new ExchangeError ('Not Implemented');
+    async commit (symbol = '', balance = '', market = '', params = {}) {
+        let limit = this.safeFloat (this.channelLimits, symbol);
+        if (limit < parseFloat (balance)) {
```

Comment:
We don't want to give them an error that says the limit is too big? I guess I have always felt weird about changing the amount they want to commit without confirmation that this is what they want to do?

Reply:
This was updated. we now throw an error if the balance is above the limit